### ISSUE TITLE
Plans for support checkra1n v0.12.01

### ISSUE THREAD

**Issue by wfojas** [foxlet/bootra1n#118]
Curious as to what your roadmap and release dates may be?

Thanks,

**Comment by hack-r** [comment]
Yes, is there any way we can update the version of checkra1n ourselves?

**Comment by cnrkuo** [comment]
https://github.com/foxlet/bootra1n/issues/127#issuecomment-753865100
